data/sales.txt

```diff
@@ -1284,6 +1284,77 @@ outfitter "sheragi"
 	"Sheragi Anti-Missile"
 	"Small EM Battery"
 	"Small Hybrid Cooling"
+shipyard "successors"
+	"Valkkyiara-Khorasaqra"
+	"Aaulqra"
+	"Ejniisde"
+	"Kaskhorade"
+	"Kvelqde"
+	"Kvelqde-Khora"
+	"Myaede"
+	"Ojde"
+	"Seiitej"
+	"Stolsaqra"
+	"Veusade"
+	"Nnesaarau"
+	"Vujlet"
+	"Nnaatej"
+	"Kijra-Khora"
+outfitter "successors"
+	"Auxiliary Power Reserve"
+	"Primary Power Reserve"
+	"Bimodal Coilgun Turret"
+	`"Ojet" Bimodal Coilgun`
+	`"Sjeja" Kinetic Lance`
+	"High-Velocity Spike"
+	"High-Velocity Spike Rack"
+	"Kinetic Rail Rifle"
+	"Phasing Laser Sidearm"
+	"Heirloom Hunter-Killers"
```

Comment:
Same for the V-K's heirloom outfits---they won't be included in the currently-open Successors PR.